seacrowd/sea_datasets/id_vaccines_tweet/id_vaccines_tweet.py

```diff
@@ -0,0 +1,127 @@
+br /&amp;gt;disalurkan melalui media sosial. Adanya respon dari masyarakat tersebut dapat menjadi&amp;lt;br
+/&amp;gt;salah satu acuan untuk melakukan evaluasi terhadap pelaksanaan vaksinasi di Indonesia.&amp;lt;br /&amp;
+gt;Maka dari itu, dengan memanfaatkan data dari media sosial twitter, penelitian ini&amp;lt;br /&amp;gt;bertujuan
+ untuk menganalisis respon masyarakat terhadap pelaksanaan vaksinasi&amp;lt;br /&amp;gt;dengan cara mengklasifikasikan
+ respon tersebut ke dalam respon positif, negatif, dan&amp;lt;br /&amp;gt;netral. Hasil analisis diperoleh bahwa
+ respon masyarakat selama 3 bulan, yaitu pada&amp;lt;br /&amp;gt;bulan Mei dan Juni masih terdapat respon netral
+ sedangkan pada bulan Juli keseluruhan&amp;lt;br /&amp;gt;kata memiliki respon negatif seiring dengan kenaikan
+ kasus Covid’19.&amp;lt;/p&amp;gt;
+&amp;lt;p&amp;gt;&amp;lt;br /&amp;gt;&amp;lt;strong&amp;gt;Kata Kunci:&amp;lt;/strong&amp;gt; covid’19, vaksinasi,
+pelaksanaan vaksin, kartu/sertifikat vaksin, sentimen&amp;lt;br /&amp;gt;analisis&amp;lt;/p&amp;gt;}, number={2},
+journal={Khazanah: Jurnal Mahasiswa}, author={Febriyanti, Syintya and Nursidah, Dea Ratu and Gustiara, Dela and
+Yulianti, Rika}, year={2022}, month={May} }
+"""
+
+_DESCRIPTION = """\
+Dataset containing tweets about COVID-19 vaccines with manually labelled information about whether they are a
+subjective tweet and their sentiment polarity. Tweets are from 20-27 June 2021 and 15-22 July 2021.
+"""
+
+_HOMEPAGE = "	https://github.com/rayendito/id-vaccines-tweets"
+
+_LICENSE = "UNKNOWN"
+
+_URL = "https://raw.githubusercontent.com/rayendito/id-vaccines-tweets/main/id_vaccines_tweets.csv"
+
+_SUPPORTED_TASKS = [Tasks.SENTIMENT_ANALYSIS]
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "0.0.1"
+
+
+class id_vaccines_tweetDataset(datasets.GeneratorBasedBuilder):
+    """Dataset containing tweets about COVID-19 vaccines with manually labelled information about whether they are
+    a subjective tweet and their sentiment polarity. Tweets are from 20-27 June 2021 and 15-22 July 2021."""
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name="id_vaccines_tweet_source",
+            version=datasets.Version(_SOURCE_VERSION),
+            description="Dataset containing tweets about COVID-19 vaccines with manually labelled information about " "whether they are a subjective tweet and their sentiment polarity. " "Tweets are from 20-27 June 2021 and 15-22 July 2021.",
+            schema="source",
+            subset_id="id_vaccines_tweet",
+        ),
+        SEACrowdConfig(
+            name="id_vaccines_tweet_seacrowd_text",
+            version=datasets.Version(_SOURCE_VERSION),
+            description="Dataset containing tweets about COVID-19 vaccines with manually labelled information about " "whether they are a subjective tweet and their sentiment polarity. " "Tweets are from 20-27 June 2021 and 15-22 July 2021.",
+            schema="seacrowd.text",
```

Comment:
Let's have the `schema name` following the format
schema="seacrowd_[seacrowd_schema_name]",

So, in this case it would be 
```suggestion
            schema="seacrowd_text",
```